src/lib/DepositRecordSerializer.js

```diff
@@ -79,7 +80,19 @@ export class DepositRecordSerializer {
   */
   serializeContributors(record) {
     const in_contributors = record.contributors || [];
-    const contributors = in_contributors.map((contributor) => {
+
+    // Remove contributors with only a type
+    // Note: we have to do this because type is filled by default, but
+    // contributors is an optional field
+    let contributors = in_contributors.filter((contributor) => {
```

Comment:
This is more of what you mentioned @zzacharo :disappointed: but I don't know of a good alternative for default values. 

We are in this situation where we want a default value to be shown AND stored IF other fields of the contributor components are filled... Otherwise we would still need to do fiddly serialization work where we inject the default value if contributors has other values which is the reverse  of what we doing here: removing the default value if contributors has no other values  

Other ideas? Maybe Formik has a way to deal with that kind of problem?